crates/holochain/src/core/workflow/incoming_dht_ops_workflow/incoming_ops_batch.rs

```diff
@@ -118,7 +118,7 @@ mod tests {
         let (mut batch_entry, first_recv) = batch.check_insert(false, vec![]);
         assert!(batch.is_running());
 
-        let handle = tokio::spawn({
+        tokio::spawn({
```

Comment:
I've fixed some warnings here. We have a bunch at the moment so it's easy for new ones to slip in. I've fixed the ones I've let slip through recently here

Reply:
yeah... we used to deny warnings. I don't actually remember having a discussion agreeing to allow warnings again. One way to clean these up would be do deny them again : )

Reply:
I might also suggest we actually manage this, and all tasks, instead of throwing out the handle, at least with `TaskManagerClient::add_cell_task_ignored`. We want to be managing all tasks, so they all shutdown when the conductor shuts down. Maybe we can at least track these with TODOs when we find them.

Reply:
This one is in test code so I'm not too worried about the shutdown. If I understand right there is a Tokio runtime created per test so the runtime itself will shutdown after the test

Reply:
Denying warnings in code is a pain if somebody builds with different feature flags or a newer clippy version and the code won't compile. If we could just disallow warnings in CI that'd be great